### ISSUE TITLE
Keybase APP won't start

### ISSUE THREAD

**Issue by Essque** [keybase/client#9335]
The background processes seem to be fine, for instance keybase id works ok.

Two issues I can see:
- The Keybase App doesn't start
- The /keybase folder won't mount / can't be created

my log id: 67c2f3fc3bfe8bdde6dc851c

**Comment by ineptech** [comment]
Same here.  GUI app won't open, keybase id does work, keybase folder won't mount, but I can push to a git via keybase.  Win7.  No problems prior to a week or so.  log id 6211dcc6650e470c5f04c61c

**Comment by maxtaco** [comment]
@zanderz  can you please take a look?

**Comment by zanderz** [comment]
@Essque That log send is not from a Windows machine, is that the right one?

@ineptech I understand you can click the tray icon, but nothing pops up, is that right? We have a few users reporting such on Win7 but haven't reproduced. Did you recently have a second monitor plugged in?

**Comment by Essque** [comment]
Hey @zanderz that's the right one. ineptech appears to be on Windows, I'm on Mac. 
Cheers for having a look into it, it's much appreciated.

**Comment by ineptech** [comment]
@zanderz Yes, you got it.  I see the tray icon, clicking on it refreshes the taskbar icon but does not open the client app, and I do indeed have two monitors.  Disabling (via windows settings) one display and restarting keybase did not affect the issue.  Here's a log with one display disabled on the offchance it's useful: 7ef42106ce389cfb768be11c

Let me know how I can help troubleshoot.
